client/related_items/ItemCard.jsx

```diff
@@ -46,6 +47,15 @@ class ItemCard extends React.Component {
             });
           });
       });
+
+    getProductRatings(this.props.id)
+      .then(ratingsMeta => {
+        let averageRating = computeAverageRating(ratingsMeta.ratings)[1];
```

Comment:
Please let me know if the [1] indexing is not very readable! I plan to refactor a lot of my code next week and readability is definitely going to be my top-priority. I can change the helper to return an object instead for a more readable access.

Reply:
Haha I just had to look at your function and understood right away. Agreed that eventually an object would make this more readable - let me know if/when you do that so I can update how I'm using the function on my end as well!